src/main/java/service/FruitReadFile.java

```diff
@@ -0,0 +1,22 @@
+package service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FruitReadFile implements ReadFile {
```

Comment:
```suggestion
public class FileReaderImpl implements FileReader {
```
we should use nouns for class/interface names